device/vxlan/destinationipmode.yaml

```diff
@@ -11,6 +11,21 @@ components:
           enum: [unicast, multicast]
           default: unicast
         unicast:
-          $ref: './unicast.yaml#/components/schemas/Vxlan.Unicast'
+          $ref: './unicast.yaml#/components/schemas/Vxlan.V4Tunnel.DestinationIPMode.Unicast'
```

Comment:
All the properties within unicast are as array.
```
unicast : [
remote_vtep_address,
suppress_arp,
remote_vm_mac,
remote_vm_ipv4
]
```

Reply:
updated